src/js/components/Stack/StyledStack.js

```diff
@@ -2,6 +2,7 @@ import styled from 'styled-components';
 
 const StyledStack = styled.div`
   position: relative;
+  display: flex;
```

Comment:
This seems odd to me, since all of the children except one are position absolute. Why was this added?